akka-cluster-client-grpc-scala/README.md

```diff
@@ -0,0 +1,49 @@
+# Cluster Client with gRPC transport
+
+The purpose of this example is to illustrate how the deprecated
+[Akka Cluster Client](https://doc.akka.io/docs/akka/2.5/cluster-client.html)
+can be replaced by [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html).
+
+The example is intended to be copied and adjusted to your needs. It will not be
+provided as a published artifact.
+
+The example is still using an actor on the client side to have an API that is very close
+to the original Cluster Client. The messages this actor can handle correspond to the
+[Distributed Pub Sub](https://doc.akka.io/docs/akka/current/distributed-pub-sub.html)
+messages on the server side, such as `ClusterClient.Send`.
+
+The `ClusterClient` actor delegates those messages to the gRPC client and on the
+server side those are translated and delegated to the destination actors that
+are registered via the `ClusterClientReceptionist` in the same way as in the original.
+
+The application specific messages are wrapped and serialized with Akka serialization.
+
+A more decoupled, and therefore better, solution would be to use Akka gRPC directly
+and define an application specific protocol buffer messages and gRPC service calls.
+Then the clients don't even have to be using Akka. If you are building a new
+application that is recommended, but if you are already using Akka Cluster Client
+the approach illustrated here is a good migration path that requires minimal changes
+to existing source code.
```

Comment:
Perhaps explicitly state that the server and client does not need to communicate using remoting/cluster at _all_ anymore using this solution somewhere in this block, just for good measure.